.travis.yml

```diff
@@ -1,8 +1,6 @@
 sudo: false
 language: node_js
 env: TEST_SUITE=unit
-node_js:
-  - "5.6.0"
```

Comment:
We can add a comment too saying that the node version is handled in `.nvmrc`. 

Also most of the time travis breaks it will be something else plus travis already shows the version in the log & also that it picks it up from `nvm`. 

I'm planning to make the tools & build process more robust so that things doesn't break unless it's something major.